app/pages/clinicworkspace/ClinicianPatients.js

```diff
@@ -57,15 +57,16 @@ export const ClinicianPatients = (props) => {
   const [showDeleteDialog, setShowDeleteDialog] = useState(false);
   const [showAddPatientDialog, setShowAddPatientDialog] = useState(false);
   const [showEditPatientDialog, setShowEditPatientDialog] = useState(false);
-  const [showNames, setShowNames] = useState(false);
-  const [searchText, setSearchText] = React.useState('');
   const [page, setPage] = useState(1);
   const [pageCount, setPageCount] = useState();
   const [selectedPatient, setSelectedPatient] = useState(null);
   const [loading, setLoading] = useState(false);
   const [patientFormContext, setPatientFormContext] = useState();
   const rowsPerPage = 8;
 
+  const patientListSearchTextInput = useSelector(({ blip }) => blip.patientListFilters.patientListSearchTextInput);
+  const isPatientListVisible = useSelector(({ blip }) => blip.patientListFilters.isPatientListVisible);
```

Comment:
Just curious about why this is split across two `useSelector` calls instead of the object deconstruction in `ClinicPatients`

Reply:
No real reason, just my inattention. I believe that splitting into multiple useSelector calls is technically better to for reduce re-rendering when returning primitives, but I think in this case we consume both values in the state so it doesn't really matter.

Unified here: a0c47bdd69b300108edb1f6ba9d717e9ff06b9d6